<html>
  <style>
    div {
width: 70%;
height: 70%;
background:white;
border-radius: 20px;
font-family: courier, serif;
}
    .center {
  margin: auto;
  background: white;
  width: 80%;
}
    
      </style>
    <div class="center">
    <body>

<h1 id="protocols"><strong>PROTOCOLS</strong></h1>
<h1 id="table-of-contents"><strong>Table of contents</strong></h1>
<h2 id="molecular-biology"><a href="#1-molecular-biology-1"><strong>1.
Molecular biology</strong></a></h2>
<h3 id="dna"><a href="#11-dna-1"><strong>1.1 DNA</strong></a></h3>
<h4 id="genomic-dna-from-qiagen-kit"><a
href="#111-genomic-dna-from-qiagen-kit-1">1.1.1. Genomic DNA from QIAGEN
kit</a></h4>
<h4
id="swarms-dna-extraction-suitable-for-both-long-and-short-read-seq"><a
href="#112-swarms-dna-extraction-suitable-for-both-long-and-short-read-seq-1">1.1.2.
Swarms DNA extraction (suitable for both Long and Short-Read
seq)</a></h4>
<h4 id="chelex"><a href="#113-chelex-1">1.1.3. Chelex</a></h4>
<h4 id="spore-dna-extraction-suitable-for-both-short-read-seq"><a
href="#114-spore-dna-extraction-suitable-for-both-short-read-seq-1">1.1.4.
Spore DNA extraction (suitable for both Short-Read seq)</a></h4>
<h4 id="fish"><a href="#115-fish-1">1.1.5. FISH</a></h4>
<h4 id="colony-pcr"><a href="#116-colony-pcr-1">1.1.6. Colony
PCR</a></h4>
<h3 id="rna"><a href="#12-rna-1"><strong>1.2 RNA</strong></a></h3>
<h4 id="total-rna-extraction"><a
href="#121-total-rna-extraction-1">1.2.1. Total RNA extraction</a></h4>
<h4 id="northern-blot"><a href="#122-northern-blot-1">1.2.2. Northern
blot</a></h4>
<h3 id="protein"><a href="#13-protein-1"><strong>1.3
Protein</strong></a></h3>
<h4 id="quick-extraction"><a href="#131-quick-extraction-1">1.3.1. Quick
extraction</a></h4>
<h4 id="acidic-extraction"><a href="#132-acidic-extraction-1">1.3.2.
Acidic extraction</a></h4>
<h4 id="western-blot"><a href="#133-western-blot-1">1.3.3. Western
blot</a></h4>
<h3 id="transformations"><a href="#14-transformations-1"><strong>1.4
Transformations</strong></a></h3>
<h4 id="targeted-insertion"><a href="#141-targeted-insertion-1">1.4.1.
Targeted insertion</a></h4>
<h4 id="allele-swap"><a href="#142-allele-swap-1">1.4.2. Allele
swap</a></h4>
<h2 id="stocks-and-sample-preparations"><a
href="#2-stocks-and-sample-preparations-1"><strong>2. Stocks and sample
preparations</strong></a></h2>
<h3 id="freezer-stocks"><a href="#21-freezer-stocks-1"><strong>2.1
Freezer stocks</strong></a></h3>
<h4 id="from-liquid-cultures"><a
href="#211-from-liquid-cultures-1">2.1.1. From Liquid Cultures</a></h4>
<h4 id="from-solid-cultures"><a href="#212-from-solid-cultures-1">2.1.2.
From Solid Cultures</a></h4>
<h3 id="microscopy"><a href="#22-microscopy-1"><strong>2.2
Microscopy</strong></a></h3>
<h4 id="mutliple-long-time-lapses-with-plate-skirts"><a
href="#221-mutliple-long-time-lapses-with-plate-skirts-1">2.2.1.
Mutliple Long Time Lapses with Plate-Skirts</a></h4>
<h4 id="single-cell-imaging"><a href="#222-single-cell-imaging-1">2.2.2.
Single cell imaging</a></h4>
<h4 id="morphological-characterization"><a
href="#223-morphological-characterization-1">2.2.3. Morphological
characterization</a></h4>
<h2 id="assays"><a href="#3-assays-1"><strong>3.
Assays</strong></a></h2>
<h3 id="competition-assays"><a
href="#31-competition-assays-1"><strong>3.1. Competition
assays</strong></a></h3>
<h4 id="assay-using-liquid-cultures"><a
href="#311-assay-using-liquid-cultures-1">3.1.1. Assay using liquid
cultures</a></h4>
<h4 id="assay-using-plates"><a href="#312-assay-using-plates-1">3.1.2.
Assay using plates</a></h4>
<h4 id="mixing-incompatibility-estimate-and-quantification"><a
href="#42-myxobacteria-isolation-from-soil-1">3.1.3. Mixing
Incompatibility Estimate and Quantification</a></h4>
<h3 id="predation-assays"><a href="#32-predation-assays-1"><strong>3.2
Predation assays</strong></a></h3>
<h4 id="spot-assays"><a href="#321-spot-assays-1">3.2.1. Spot
assays</a></h4>
<h4 id="lawn-assay"><a href="#322-lawn-assay-1">3.2.2. Lawn
assay</a></h4>
<h4 id="predatory-behaviours-at-single-cell-level"><a
href="#323-predatory-behaviours-at-single-cell-level-1">3.2.3. Predatory
behaviours at single cell level</a></h4>
<h2 id="other-protocols"><a href="#4-other-protocols-1"><strong>4. Other
protocols</strong></a></h2>
<h3 id="aggregates-isolation"><a
href="#41-aggregates-isolation-1"><strong>4.1. Aggregates
isolation</strong></a></h3>
<h3 id="myxobacteria-isolation-from-soil"><a
href="#42-myxobacteria-isolation-from-soil-1"><strong>4.2. Myxobacteria
Isolation from Soil</strong></a></h3>
<h4 id="prey-baits"><a href="#421-prey-baits-1">4.2.1. Prey
baits</a></h4>
<h4 id="isolation-of-myxococcaceae"><a
href="#422-isolation-of-myxococcaceae-1">4.2.2. Isolation of
<em>Myxococcaceae</em></a></h4>
<hr />
<h1 id="molecular-biology-1"><strong>1. Molecular biology</strong></h1>
<h2 id="dna-1"><strong>1.1 DNA</strong></h2>
<h3 id="genomic-dna-from-qiagen-kit-1"><strong>1.1.1. Genomic DNA from
QIAGEN kit</strong></h3>
<p>For detailed protocol, refer to the handbook <em>QIAGEN Genomic DNA
Handbook</em></p>
<p><strong>NB:</strong> <em>This protocol works well with cells from
liquid cultures. However, it does not work with spores nor with cells at
high densities harvested directly from plates.</em></p>
<ol type="1">
<li>Add 2 µl of RNase A (100 mg/ml) to 1 ml of Buffer B1.</li>
<li>Pellet cells at 12000 rpm for 5 min and discard supernatant.</li>
<li>Resupsend bacteria pellet in 1 ml of Buffer B1 with RNase. Gently
pippette up and down and then invert tube without vortexing.</li>
<li>Add 80 µl of Lysozyme (100 mg/ml) and 45 µl of Prot K (20 mg/ml).
Incubate for at least 45 min at 37 °C while roughly every 10 min gently
invert the tube - DO NOT VORTEX.</li>
<li>Transfer solution to a 15 ml falcon tube and add 1.2 ml of Buffer B2
and mix by inverting the tube multiple times. Incubate at 50 °C for 30
min.</li>
<li>Place Genomic-tip 20/G column on vaccum pump holder (previously
washed and dried). Equilibrate column with 1 ml of QBT Buffer and allow
column to empty by gravity flow.</li>
<li>Invert tube containing sample multiple times and apply to column.
Allow to enter the resin by gravity flow or apply vaccum (do not exceed
10 kPa).</li>
<li>Wash column 3x with 1 ml of QC Buffer. Try to maintain column resin
humid and do not leave it to dry.</li>
<li>Elute DNA with 2 ml of QF Buffer preheated at 50 °C. Typically this
step run O/N gives the best yields.</li>
<li>Precipitate DNA by adding 1.2 ml of isopropanol. Gently invert tube
20 times.</li>
<li>Spool DNA with glass probes if the starting material was
sufficiently aboundant (DNA is visibile with neked eye). Otherwise
continue form step 13.</li>
<li>Transfer pooled DNA in TE or EB buffer depending on following
application.</li>
<li>Centrifuge at 4°C at 9000 rcf for 10 min.</li>
<li>Remove supernatant, add cold Et-OH 70 % and centrifuge again at 9000
rcf for 10 min. Gently discard supernatant and let air dry for about 15
min. Resuspend pellet in TE or EB buffer (typically 50 µl) depending on
the following applicaiton. ### <strong>1.1.2. Swarms DNA extraction
(suitable for both Long and Short-Read seq)</strong></li>
</ol>
<p><em>This protocol is based on the PowerBiofilm QIAGEN kit.</em><br />
<strong>NB for long-read seq:</strong> <span style="color:red">Instead
of <strong>gently invert tubes</strong> to preserve long DNA
fragments.</span> - Preheat MBL media provided in the kit at 55 °C. 1.
Spin down cells at 14000 rpm. 2. Remove supernatant and add 350 µl of
MBL media. Transfer to the PowerBiofilm Tube. 3. Add 100 µl of FB
solution and vortex briefly. 4. Incubate PowerBiofilm Bead Tube at 65 °C
for 5 min. 5. Use Bead Ruptor at GDC: - 1 Cycle - 30 sec - 5.30 power (~
3200 rpm) 6. Centrifuge at 14000 rpm for 1 min. Transfer supernatant in
a 2 ml tube. 7. Add 100 µl of IRS solution and store at 4 °C for 5 min.
8. Centrifuge at 14000 rpm for 1 min at RT. 9. Avoiding the pellet,
transfer all supernatant to a 2 ml. Then add 900 µl of MR solution and
vortex briefly. 10. Load 650 µl of supernatant to a MB Spin-Column and
centrifuge at 14000 rpm for 1 min at RT. Discard flow-through and repeat
until the all supernatant had been processed. 11. Place MB Spin-Col into
a new 2 ml collection tube. 12. Add 650 µl of PW solution and centrifuge
at 14000 rpm for 1 min. 13. Discard flow-through and add 650 µl of Et-OH
and centrifuge at 14000 rpm for 1 min. 14. Discard flow-through and
centrifuge at 14000 rpm for 2 min again. 15. Place the MB Spin-Column in
a new 2 ml collection tube. 16. Add 100 µl of EB solution to the center
of the column. 17. Cetrifuge at 14000 rpm for 1 min. 18. Discard column
and store the DNA at 4 °C or -20 °C according to application.</p>
<h3 id="chelex-1"><strong>1.1.3. Chelex</strong></h3>
<ol type="1">
<li>Spin down cells at 12000 rpm for 5 min.</li>
<li>Remove supernatant and add 250 µl of Chelex-TE solution pH 8.0.
Resuspend pellet by pippetting not too vigorously to avoid DNA
shearing.</li>
<li>Incubate samples for 25 min at 80 °C.</li>
<li>Heat at 90 °C for 10 min.</li>
<li>Incubate in ice for 5 min.</li>
<li>Spin down at 13000 rpm for 3 min.</li>
<li>Remove supernatant.</li>
<li>Store at 4 °C or -20 °C according to following procedure.</li>
</ol>
<h3
id="spore-dna-extraction-suitable-for-both-short-read-seq-1"><strong>1.1.4.
Spore DNA extraction (suitable for both Short-Read seq)</strong></h3>
<h3 id="fish-1"><strong>1.1.5. FISH</strong></h3>
<h3 id="colony-pcr-1"><strong>1.1.6. Colony PCR</strong></h3>
<h2 id="rna-1"><strong>1.2 RNA</strong></h2>
<h3 id="total-rna-extraction-1">1.2.1. Total RNA extraction</h3>
<h3 id="northern-blot-1">1.2.2. Northern blot</h3>
<h2 id="protein-1"><strong>1.3 Protein</strong></h2>
<h3 id="quick-extraction-1">1.3.1. Quick extraction</h3>
<h3 id="acidic-extraction-1">1.3.2. Acidic extraction</h3>
<h3 id="western-blot-1">1.3.3. Western blot</h3>
<h2 id="transformations-1"><strong>1.4 Transformations</strong></h2>
<h3 id="targeted-insertion-1">1.4.1. Targeted insertion</h3>
<h3 id="allele-swap-1">1.4.2. Allele swap</h3>
<h1 id="stocks-and-sample-preparations-1"><strong>2. Stocks and sample
preparations</strong></h1>
<h2 id="freezer-stocks-1"><strong>2.1 Freezer stocks</strong></h2>
<h3 id="from-liquid-cultures-1"><strong>2.1.1. From Liquid
Cultures</strong></h3>
<ul>
<li>Label all cryotubes needed before starting with the following
protocol.</li>
<li>Grow cells to mid-log phase.
<ul>
<li>In case of healty and dense culture, in same cases you can avoid
measuring the OD.</li>
</ul></li>
<li>Spin down for 15 min at 5000 rpm (large centrifuge) the entire
growing culture.</li>
<li>Discard supernatant and add 2000 µl of CTT to resuspend pellet.
<ul>
<li>in case of back up, add 2500 µl.</li>
</ul></li>
<li>Once resupsended transfer 1200 µl to an individual cryotube and add
400 µl of 80% Glycerol (final conentration 20%).
<ul>
<li><strong>NOTE</strong>: Glycerol final conctration is species
specific. Most of myxo are fine with 20 %. However others (including
Mxx) can be frozen with 5% glycerol (Garcia and Müller 2016).</li>
<li>Dispense in two tubes if preparing also backup</li>
</ul></li>
<li>Before placing individual tubes in freezer stock boxes, handshake
tubes vigorously (Glycerol and CTT can separate in two pahses). ###
2.1.2. From Solid Cultures ## <strong>2.2 Microscopy</strong> ### 2.2.1.
Mutliple Long Time Lapses with Plate-Skirts ### 2.2.2. Single cell
imaging ### 2.2.3. Morphological characterization # <strong>3.
Assays</strong> ## <strong>3.1. Competition assays</strong> ### 3.1.1.
Assay using liquid cultures ### 3.1.2. Assay using plates ### 3.1.3.
Mixing Incompatibility Estimate and Quantification ## <strong>3.2
Predation assays</strong> ### 3.2.1. Spot assays ### 3.2.2. Lawn assay
### 3.2.3. Predatory behaviours at single cell level # <strong>4. Other
protocols</strong> ## <strong>4.1. Aggregates isolation</strong> ##
<strong>4.2. Myxobacteria Isolation from Soil</strong> ### 4.2.1. Prey
baits ### 4.2.2. Isolation of <em>Myxococcaceae</em></li>
</ul>
    </div>
    </body>
</html>
